### PR TITLE
Adding pvcaptest package

### PR DESCRIPTION
**Pull request by bt-**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.

Yes, I am submitting the pull request and the only maintainer listed.

### CODE REVIEW

File: recipes/pvcaptest/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    - numpy >=1.13.0
+    - pandas >=0.24
+    - pip
+    - setuptools
+    - python >=3.5
+    - python-dateutil >=2.5
+    - scikit-learn >=0.19
+    - statsmodels >=0.8
+    - holoviews >=1.11
+    - pvlib-python >=0.7
```
Comment (marcelotrevisani) [inline recipes/pvcaptest/meta.yaml:29]: I think here you just need ``python`` and ``pip`` to package this project
```suggestion
    - pip
    - python >=3.5
```

File: recipes/pvcaptest/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+  run:
+    - bokeh >=1
+    - matplotlib >=2
```
Comment (marcelotrevisani) [inline recipes/pvcaptest/meta.yaml:33]:
```suggestion
    - matplotlib-base >=2
```

File: recipes/pvcaptest/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    - python >=3.5
+    - python-dateutil >=2.5
+    - scikit-learn >=0.19
+    - statsmodels >=0.8
+    - holoviews >=1.11
+    - pvlib-python >=0.7
+
+test:
+  imports:
+    - captest
```
Comment (marcelotrevisani) [inline recipes/pvcaptest/meta.yaml:45]: Can we try to add ``pip check`` here please?
```suggestion
    - captest
  requires:
    - pip
  commands:
    - pip check
```

File: recipes/pvcaptest/LICENSE.txt [hunk 4]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (marcelotrevisani) [inline recipes/pvcaptest/LICENSE.txt:1]: It seems the license is already being packaged on pypi, you can remove this file then

**Comment by marcelotrevisani** [comment]
If you want to, next time you might give a go to [grayskull](https://github.com/marcelotrevisani/grayskull to generate recipes for packages which are on ``pypi``

File: recipes/pvcaptest/meta.yaml (hunk 1 above)
Comment (bt-) [reply recipes/pvcaptest/meta.yaml]: I removed the other packages from host section.

File: recipes/pvcaptest/meta.yaml (hunk 2 above)
Comment (bt-) [reply recipes/pvcaptest/meta.yaml]: I made this change to matplotlib-base.  What is the difference between these two?

File: recipes/pvcaptest/meta.yaml (hunk 3 above)
Comment (bt-) [reply recipes/pvcaptest/meta.yaml:35]: I added the pip check test.  This look to have passed when I built it locally. before committing these revisions.

File: recipes/pvcaptest/LICENSE.txt (hunk 4 above)
Comment (bt-) [reply recipes/pvcaptest/LICENSE.txt]: I removed the duplicate license file.

**Comment by bt-** [comment]
@marcelotrevisani, thank you for the review.  I think I've addressed all your comments.  I tried using grayskull through your website, which seemed to work great.  I'll keep it mind if I need to create another recipe for a package on pypi.

File: recipes/pvcaptest/meta.yaml (hunk 2 above)
Comment (marcelotrevisani) [reply recipes/pvcaptest/meta.yaml]: If you do not need the ``qt`` backend just ``matplotlib-base`` is sufficient 
``matplotlib`` will pull ``pyqt`` and the ``qt`` stack, which are a quite large dependency

**Comment by marcelotrevisani** [comment]
Nice! Thanks for adding this project to ``conda-forge``! :tada:  :tada:  :tada:  :tada:

**Pull request merged**